package.json

```diff
@@ -14,7 +14,7 @@
     "start": "tsdx watch",
     "build": "tsdx build",
     "test": "tsdx test --passWithNoTests",
-    "lint": "tsdx lint src example",
+    "lint": "eslint . --ext js,ts,jsx,tsx",
```

Comment:
Switching directly to eslint since it seemed like there were some resolution errors when we ran listing through tsdx.